### ISSUE TITLE
Make /wallets/find-wallet/ keyboard accessible

### ISSUE THREAD

**Issue by corwintines** [ethereum/ethereum-org-website#8169]
### Describe the bug

Currently, this page is not keyboard accessible. If a user lands on this page, they cannot interact with the personas, filters, or get more information for a wallet.

### To reproduce

1. Go to [find wallets](https://ethereum.org/en/wallets/find-wallet/)
2. Try to select a persona using only the keyboard
3. Try to select a filter using only the keyboard
4. Try to see more information for a wallet using only the keyboard

### Expected behavior

Users should be able to navigate selecting a persona, filter, or more wallet information using only the keyboard

### Desktop (please complete the following information)

All browsers

### Smartphone (please complete the following information)

N/A. But I imagine voice nav doesnt work either.
